internal/pkg/cli/pipeline_status.go

```diff
@@ -47,12 +61,12 @@ func BuildPipelineStatusCmd() *cobra.Command {
 	cmd := &cobra.Command{
 		Hidden: true, // TODO: remove when ready for production!
 		Use:    "status",
-		Short:  "Show the status of your application's pipeline.",
-		Long:   "Show the status of each deployed pipeline's stages.",
+		Short:  "Show the status of a pipeline.",
+		Long:   "Show the status of each stage of your pipeline.",
```

Comment:
```suggestion
		Short:  "Shows the status of a pipeline.",
		Long:   "Shows the status of each stage of your pipeline.",
```
Show**s** to be consistent with other "show" commands